jetty-server/src/test/java/org/eclipse/jetty/server/HttpOutputTest.java

```diff
@@ -852,7 +853,7 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     @Test
     public void testEmptyBufferKnown() throws Exception
```

Comment:
The current test name is weird.
Can you rename the test to be `testResponseIsCommittedWithZeroContentLength()` or similar?

Reply:
`testResponseCommittedWithEmptyBufferAndZeroContentLength()`?